tests/installation/bootloader_uefi.pm

```diff
@@ -91,7 +91,15 @@ sub run {
     # By pressing a random key, we stop the grub timeout
     send_key 'backspace' if (is_jeos && is_aarch64);
 
-    assert_screen([qw(bootloader-shim-import-prompt bootloader-grub2)], $bootloader_timeout);
+    my @needles = qw(bootloader-grub2 bootloader-shim-import-prompt);
```

Comment:
That is not good code style, to remove things that you don't need to add in the first place. Those two command grep do not compensate to avoid the `else` statement. what do you think about this?

```
    if (get_var('VERSION') =~ /agama/) {
     # comment
      assert_screen(\@needles, 'bootloader-grub2-agama');
    }
    else {
      assert_screen([qw(bootloader-shim-import-prompt bootloader-grub2)], $bootloader_timeout);
    }
```

Reply:
Updated.